sigle/src/modules/analytics/stats/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { defaultStyles } from '@visx/tooltip';
```

Comment:
as the utils are just for the tooltip should we rename it to tooltip.ts? or tooltipUtils?